Automattic/wp-calypso
# Calypso Green: add links to footer menu

See related issue

## Proposed Changes

* This PR updates the footer in Calypso Green to include links to the Affiliates program, Webinars, and testimonials pages.

|Before|After|
| ---|---|
|<img width="1165" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/3549b8db-db17-423e-b8b7-a8c0f1af691e">|<img width="1206" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/6357cfb3-6db7-4d4a-8f7f-24af545dd802">|

## Why are these changes being made?


* This link is missing, so we are adding it to both jetpack.com and cloud.jetpack.com

## Testing Instructions



* Open the Calypso live link
* Visit `/pricing`
* Scroll to the bottom of the page until you see the footer
* Make sure "For Affiliates" is one of the links under "Partners", and "Webinars" and "Reviews and Testimonials" under "Help"

## Pre-merge Checklist
